src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+          method="post" 
+          class="search__form"
+          data-qa="big"
+    >
+      <label>
```

Comment:
```suggestion
    <form 
      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
      method="post" 
      class="search__form"
      data-qa="big"
    >
      <label>
```

виправ відступи, будьласка, якщо атрибути переносяться, то переносимо починаючи з першого і відступ має бути 2 пробіли, як у вкладених елементах
чекліст, пункти 4, 5 https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md

<img width="1112" alt="image" src="https://user-images.githubusercontent.com/102413088/183249259-63c6a171-0d3e-4cd6-a1dd-f6161b627563.png">

ну і перевір це у всьому документі